samples/snippets/src/test/java/pubsub/SchemaIT.java

```diff
@@ -179,7 +265,8 @@ public void testSchema() throws Exception {
 
     bout.reset();
     // Test receiving BINARY-encoded Avro records.
-    SubscribeWithAvroSchemaExample.subscribeWithAvroSchemaExample(projectId, avroSubscriptionId);
```

Comment:
I think this makes it such that `SubscribeWithAvroSchemaExample` is not being tested. Should we create a separate test to test this?